src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileFields.scala

```diff
@@ -1,14 +1,15 @@
 package uk.gov.nationalarchives.tdr.api.graphql.fields
 
-import java.util.UUID
 import io.circe.generic.auto._
 import sangria.macros.derive._
 import sangria.marshalling.circe._
 import sangria.schema.{Argument, Field, InputObjectType, ListType, ObjectType, fields}
-import uk.gov.nationalarchives.tdr.api.auth.{ValidateHasExportAccess, ValidateUserHasAccessToConsignment}
+import uk.gov.nationalarchives.tdr.api.auth.ValidateUserHasAccessToConsignment
 import uk.gov.nationalarchives.tdr.api.graphql.ConsignmentApiContext
 import uk.gov.nationalarchives.tdr.api.graphql.validation.UserOwnsConsignment
-import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes._
+import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes.UuidType
```

Comment:
Think this is an unnecessary import: `import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes.UuidType`

Reply:
I've removed it.

Reply:
It is necessary, it's an implicit import that Intellij thinks isn't used but it won't compile without it. I've put it back.

Reply:
It is necessary, it's an implicit import that Intellij thinks isn't used but it won't compile without it. I've put it back.